sass/_base.sass

```diff
@@ -0,0 +1,1793 @@
+
+	&.open-nav .flyout-button
+		display: none
+
+	.logo
+		position: absolute
+		top: 40px
+		left: 50%
+		transform: translate(-50%, -50%)
+		display: block
+		width: 45px
+		height: 44px
+		background-image: url(/images/favicon.png)
+
+
+	&.flip-nav .flyout-button
+		background-image: url(/images/toc_icon_grey.png)
+
+
+.nav-buttons
+	float: right
+
+#viewDocs, #tryKubernetes
+	display: none
+
+#viewDocs
+	border: 2px solid white
+	background-color: transparent
+	transition: 0.3s
+
+	&:hover
+		background-color: white
+		color: $dark-grey
+
+#tryKubernetes
+	width: 0
+	padding: 0 0
+	border: 1px solid transparent
+	background-color: transparent
+	text-align: center
+	white-space: nowrap
+	vertical-align: middle
+	overflow: hidden
+	transition: 0.3s
+
+#hamburger
+	display: inline-block
+	position: relative
+	vertical-align: middle
+	padding: 0
+	border: 0
+	background: none
+
+	div, &:before, &:after
+		position: absolute
+		left: 15%
+		width: 70%
+		height: 2px
+		background-color: $blue
+		transition: 0.3s
+		content: ""
+
+	div
+		top: calc(50% - 1px)
+
+	&:before
+		top: 24%
+
+	&:after
+		bottom: 24%
+
+	&:hover
+		div, &:before, &:after
+			background-color: white
+
+#mainNav
+	h5
+		color: $blue
+		font-weight: normal
+
+	main
+		white-space: nowrap
+		overflow: hidden
+		clear: both
+
+	.nav-box
+		float: left
+		white-space: normal
+
+	h3
+		a
+			color: $blue
+			text-decoration: none
+
+// Global Nav - 12/9/2016 Update
+
+ul.global-nav
+	display: none
+
+	li
+		display: inline-block
+		margin-right: 14px
+
+		a
+			color: #fff
+			font-weight: 400
+			padding: 0
+			position: relative
+
+			&.active:after
+				position: absolute
+				width: 100%
+				height: 2px
+				content: ''
+				bottom: -4px
+				left: 0
+				background: #fff
+
+			.ui-icon
+				filter: brightness(0) invert(1)
+
+		ul
+			display: none
+			position: fixed
+			top: 40px
+			text-align: left
+
+			li
+				display: block
+				height: 28px
+
+				a
+					background: $dark-grey
+					color: #fff
+					padding: 7px
+
+			li:last-child
+				a
+					border-radius: 7px
+
+	li:hover
+		ul
+			display: block
+
+
+.flip-nav ul.global-nav li a,
+.open-nav ul.global-nav li a,
+	color: $dark-grey
+
+.flip-nav ul.global-nav li a .ui-icon
+	filter: brightness(0)
+
+
+.flip-nav ul.global-nav li ul li a,
+	background: #fff
+	color: $dark-grey
+
+.flip-nav ul.global-nav li a.active:after,
+.flip-nav ul.global-nav li ul li a.active:after,
+.open-nav ul.global-nav li a.active:after,
+
+	background: $blue
+
+// FLIP NAV
+.flip-nav
+	header
+		background-color: white
+
+
+// OPEN NAV
+.open-nav
+	body
+		overflow: hidden
+
+	#cellophane
+		display: block
+		z-index: 9998
+
+	header
+		background-color: #e8e8e8
+		z-index: 9999
+
+	#hamburger
+		div
+			opacity: 0
+
+		&:before, &:after
+			left: 12px
+			transform-origin: 0 1px
+
+		&:before
+			transform: rotate(45deg)
+
+		&:after
+			transform: rotate(-45deg)
+
+.open-nav, .y-enough
+	#tryKubernetes
+		width: 150px
```

Comment:
Would you accept an update here or should I continue with PR#10353?

Reply:
I would say hold off on that. Wait for this PR to be merged, then move all of the Sass files in `assets/sass` into this root-level folder in your PR. That strikes me as the least messy option.

Reply:
👍